src/app/public/plugin-resources/code-examples/summary-action-bar/modal/summary-action-bar-modal-demo.component.html

```diff
@@ -0,0 +1,76 @@
+<sky-modal>
+  <sky-modal-header>
+    Summary action bar in modal
+  </sky-modal-header>
+  <sky-modal-content>
+    Hello world!
+  </sky-modal-content>
+  <sky-modal-footer>
+    <sky-summary-action-bar>
+      <sky-summary-action-bar-actions>
+        <sky-summary-action-bar-primary-action
+          (actionClick)="onPrimaryActionClick()"
+        >
+          Primary action
+        </sky-summary-action-bar-primary-action>
+        <sky-summary-action-bar-secondary-actions>
+          <sky-summary-action-bar-secondary-action
+            (actionClick)="onSecondaryActionClick()"
+          >
+            Secondary action
+          </sky-summary-action-bar-secondary-action>
+          <sky-summary-action-bar-secondary-action
+            (actionClick)="onSecondaryAction2Click()"
+          >
+            Secondary action 2
+          </sky-summary-action-bar-secondary-action>
+        </sky-summary-action-bar-secondary-actions>
+        <sky-summary-action-bar-cancel
+          (actionClick)="onCancelClick()"
+        >
+          Cancel
```

Comment:
Should this button be wired up to close the modal like in the demo? When I open in StackBlitz, nothing happens when I select Cancel.

Reply:
This should be fixed in my latest push.